### PR TITLE
Initialize symfony project

### LINKED ISSUE #1: Initialize project
Prepare small, raw Symfony application for the Hub purposes.

### PR DESCRIPTION
**Pull request by tuka217**
Q	|A
----|------
Bug fix?|	no
New feature?|	yes
BC breaks?|	no
Fixed tickets|	resolve #1
License|	MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+    "name": "intern/hub",
```
Comment (pamil) [inline composer.json]: sylius/hub?

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "license": "proprietary",
+    "type": "project",
+    "autoload": {
+        "psr-4": {
+            "Sylius\\Hub\\": "src/Sylius/Hub"
```
Comment (pamil) [inline composer.json]: Missing trailing slash at the end of path.

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests/"
+        },
+        "files": [
+            "vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php"
+        ]
+    },
+    "require": {
+        "php": ">=5.5.9",
```
Comment (pamil) [inline composer.json]: Why not PHP 7.0+ / 7.1+?

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            "app/AppKernel.php",
+            "app/AppCache.php"
+        ]
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests/"
+        },
+        "files": [
+            "vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php"
```
Comment (pamil) [inline composer.json]: Do we really need it?

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            "Sylius\\Hub\\": "src/Sylius/Hub"
+        },
+        "classmap": [
+            "app/AppKernel.php",
+            "app/AppCache.php"
+        ]
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests/"
```
Comment (pamil) [inline composer.json]: `Tests\\` -> `Tests\\Sylius\\Hub\\`?

File: phpunit.xml.dist [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+        <server name="KERNEL_DIR" value="app/" />
+    </php>
+
+    <testsuites>
+        <testsuite name="Project Test Suite">
+            <directory>tests</directory>
+        </testsuite>
+    </testsuites>
+
+    <filter>
```
Comment (pamil) [inline phpunit.xml.dist]: This node is probably not used now.

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,19 @@
+/app/config/parameters.yml
+/build/
+/phpunit.xml
+/var/*
+!/var/cache
+/var/cache/*
+!var/cache/.gitkeep
```
Comment (pamil) [inline .gitignore]: `!var` -> `!/var`

File: .gitignore [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+!/var/logs
+/var/logs/*
+!var/logs/.gitkeep
```
Comment (pamil) [inline .gitignore]: `!var` -> `!/var`

File: .gitignore [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+!/var/sessions
+/var/sessions/*
+!var/sessions/.gitkeep
```
Comment (pamil) [inline .gitignore]: `!var` -> `!/var`

File: .gitignore [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+!var/SymfonyRequirements.php
```
Comment (pamil) [inline .gitignore]: `!var` -> `!/var`

File: .gitignore [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+/vendor/
+/web/bundles/
+/var/logs
+/var/cache
+/var/sessions
```
Comment (pamil) [inline .gitignore]: Missing blank line.
Anyway, can we sort this list alphabetically?

File: app/AppKernel.php [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        return __DIR__;
+    }
+
+    public function getCacheDir()
+    {
+        return dirname(__DIR__).'/var/cache/'.$this->getEnvironment();
+    }
+
+    public function getLogDir()
```
Comment (pamil) [inline app/AppKernel.php]: Do we really need to overwrite `getRootDir`, `getCacheDir` and `getLogDir` methods?

File: app/config/services.yml [hunk 13]
```diff
@@ -0,0 +1,9 @@
+# Learn more about services, parameters and containers at
+# http://symfony.com/doc/current/service_container.html
+parameters:
```
Comment (pamil) [inline app/config/services.yml]: This file will cause application to fail.

File: app/AppKernel.php (hunk 12 above)
Comment (tuka217) [inline app/AppKernel.php]: Not sure about `getRootDir`, but the rest should be overwritten.

File: composer.json [hunk 14]
```diff
@@ -0,0 +1,67 @@
+{
+    "name": "sylius/hub",
+    "license": "proprietary",
+    "type": "project",
```
Comment (TheMadeleine) [inline composer.json]: missing `description`, and the build fails :)

File: .gitignore [hunk 15]
```diff
@@ -0,0 +1,15 @@
+/app/config/parameters.yml
+/build/
+phpunit.xml
```
Comment (pamil) [inline .gitignore]: It also matches `/foo/bar/phpunit.xml`, should be `/phpunit.xml`

**Comment by TheMadeleine** [comment]
Thanks @tuka217 ! 😃

**Pull request merged**